images/libvirtd/libvirtd-ds.yaml

```diff
@@ -12,40 +12,21 @@ spec:
     spec:
       hostNetwork: true
       hostPID: true
+      hostIPC: true
       securityContext:
         runAsUser: 0
       containers:
-        - name: daemon
+        - name: libvirtd
           image: kubevirt/libvirtd:latest
           ports:
             - containerPort: 16509
           securityContext:
             privileged: true
             runAsUser: 0
           volumeMounts:
-            - mountPath: /etc/libvirt
-              name: etc
-            - mountPath: /var/run/libvirt
```

Comment:
At least keep /var/lib/libvirt, otherwise the vms get lost after a restart and virt-handler can't see them anymore.